src/primeFactors.js

```diff
@@ -1,6 +1,15 @@
 /*
   A função primeFactors recebe um número inteiro ${number}
-  e retorna uma lista com todos os fatores desse parâmetro
+  e retorna uma lista com todos os fatores primos desse parâmetro
+
+  Por exemplo:
+
+  - os fatores primos para o número 15 são: [3, 5], pois 3 * 5 = 15
```

Comment:
Does it make sense to explicitly tell them that:

```suggestion
  - os fatores primos para o número 15 são: [3, 5], pois 3 * 5 = 15 e ambos 3 e 5 são números primos
```

?

Reply:
Yeah, better be more explicit than explicit!